raspbian/applications/rpi-update.md

```diff
@@ -0,0 +1,39 @@
+# rpi-update
+
+`rpi-update` is a command line application that will update your Raspbian kernel and Videocore firmware to the latest pre-release versions.
+
+**WARNING: Pre-release versions of software are not guaranteed to work. You should not use `rpi-update` on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process**
+
+The `rpi-update` script is supplied by a third party, "Hexxeh", and also supported by Raspberry Pi engineers. The script source can be found on Github at https://github.com/Hexxeh/rpi-update
+
+## What it does
+
+`rpi-update` will download the latest pre-release version of the linux kernel, it's matching modules, device tree files, along with the latest versions of the Videocore firmware. It will then install these files to the boot partition of the SD card, overwriting any previous versions. 
```

Comment:
Videocore -> VideoCore

Reply:
Hmm, not sure about that, I've always used Videocore. Will check.

Reply:
In this repo, both capitalisations are in use, but "VideoCore" seems more common?
https://github.com/raspberrypi/documentation/search?q=videocore&type=Code

Reply:
I guess someone beed to go through the whole repo at some point. Will create an issue